conn.go

```diff
@@ -23,6 +24,7 @@ import (
 	"unicode"
 
 	"github.com/lib/pq/oid"
+	"github.com/spaolacci/murmur3"
```

Comment:
Why do we need to hash at all?  Can't we just put the query string into the map and let its implementation take care of it?